tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -344,7 +344,7 @@ def test_find_longest_common_subsequence(self):
             },
         )
         merge = _find_timestamp_sequence(
-            [[previous_sequence, (3000, 0, 0)], [next_sequences_1, (3000, 750, 0)]],
+            [[previous_sequence, (480_000, 0, 0)], [next_sequences_1, (480_000, 120_000, 0)]],
```

Comment:
The actual timing information were not really used at this point so the tests were passing but they should not have been 😉